buildstockbatch/base.py

```diff
@@ -107,11 +107,18 @@ def path_rel_to_projectfile(self, x):
 
     def _get_weather_files(self):
         if "weather_files_path" in self.cfg:
-            logger.debug("Copying weather files")
             weather_file_path = self.cfg["weather_files_path"]
-            with zipfile.ZipFile(weather_file_path, "r") as zf:
-                logger.debug("Extracting weather files to: {}".format(self.weather_dir))
-                zf.extractall(self.weather_dir)
+            if os.path.isdir(weather_file_path):
```

Comment:
weather file path now can be directory too (not just a zipfile). Will make buildstock_local much faster if using pre-unzipped directory.

Reply:
Sweet!